src/System.Security.Permissions/ref/System.Security.Permissions.cs

```diff
@@ -0,0 +1,1108 @@
+namespace System.Security {
+  public abstract partial class CodeAccessPermission : System.Security.IPermission, System.Security.ISecurityEncodable, System.Security.IStackWalk {
+    protected CodeAccessPermission() { }
+    public void Assert() { }
+    public abstract System.Security.IPermission Copy();
+    public void Demand() { }
+    public void Deny() { }
+    public override bool Equals(object obj) { return default(bool); }
+//    public abstract void FromXml(System.Security.SecurityElement elem);
```

Comment:
The reason these are commented out is they depend on types that will be added to a different assembly as part of this larger effort (in the proposed factoring, it looks like they will land in the existing System.Security.Principal contract).

For expediency, we did not want to do cross contract changes, rather land the changes we could make and then clean up the debt.  I agree it would be good to have a bug to track.

The members commented out due to SerializatonInfo/StreamingContext and friends are in the same boat.